### DISCUSSION TITLE
source information from a person

### DISCUSSION THREAD

**Discussion by mariannevanharten** [FamilySearch/GEDCOM#357] [Q&A] [answered]
Imagine: my aunt tells me something about her parents (my grandparents).
I would enter to add my aunt as the source of this information.
My aunt is @I73@
So, I think to add:
0 @X300@ SOUR
1 NAME @I73@
That's not the coreect way.

Who can tell me the right way?

**Comment by fisharebest** [comment]
This was discussed earlier in #191

**Comment by tychonievich** [comment, marked as answer]
For this particular case:

1. NAME is defined to have a `<Text>` payload, not a `@<XREF:INDI>@`, so you can't put a pointer there. Indeed, there is no `@<XREF:INDI>@` anywhere under `SOUR`
2. Different citation standards have different idea about how to create this citation.
3. A link like this can be handled by an extension, for example as discussed in #391 but there is not currently a standard way to do this.

**Reply by dthaler** [reply]
Since the question was about "the right way", I will just add that citation guides like _Evidence Explained_ say to not cite the person per se, but rather the communication with the person.  For example EE pp. 110, 146-147 shows how to cite an interview with the person, pp.112, 153-154 shows how to cite a letter from the person, p.113, 154-155 shows how to cite an e-mail from the person, etc.  Just citing personal knowledge of someone is rare, as p.155 notes, and it goes on to say, "It is justified only when you discuss circumstances in which _you_ [emphasis added] are a firsthand witness or participant.  When you record information given to you by others, you record that as an interview, a letter, a tradition, or whatever means may be the case."

In general, based on the discussion in _Evidence Explained_, I think the person's name makes the most sense to appear in as the source's `AUTH` (not `NAME`. or not _just_ `NAME` in any case),_ and then the question becomes the same as https://github.com/FamilySearch/GEDCOM/discussions/391.
